api/render/render.go

```diff
@@ -72,12 +72,22 @@ type RenderableError interface {
 	Render(http.ResponseWriter)
 }
 
+// AsRenderableError attempts to return an error type that implements the
+// RenderableError interface.
+func AsRenderableError(err error) (RenderableError, bool) {
+	//nolint:errorlint // ignore type assertion warning. casting to interface is hard.
+	if r, ok := err.(RenderableError); ok {
+		return r, true
+	}
+	return nil, false
+}
```

Comment:
This linter is pita, is there any configuration to avoid linter error on interfaces? If not, we should add an issue to allow the skip of these cases.

Reply:
We've disabled that error, but it looks like we can enable it later. 
See https://github.com/polyfloyd/go-errorlint/issues/28#issuecomment-1254080200